setup.php

```diff
@@ -163,638 +277,1251 @@ function cloneClicked() {
+											<label class="control-label" for="iufname">Initial User's First Name:</label> <a href="#iufname_info"  class="info-anchor icon-tooltip"  data-toggle="collapse" ><i class="fa fa-question-circle" aria-hidden="true"></i></a>
+										</div>
+										<div class="col-sm-12">
+											<input name='iufname' id='iufname 'type='text' class='form-control' value='Administrator'>
+										</div>
+                                    </div>
+                                    <div id="iufname_info" class="collapse">
+                                        <a href="#iufname_info" data-toggle="collapse" class="oe-pull-away"><i class="fa fa-times oe-help-x" aria-hidden="true"></i></a>
+                                        <p>This is the First name of the 'initial user'.
+                                    </div>
+								</div>
+                            </div>
+                        </div>
+                        <div class="row">
+                            <div class="col-xs-12 ">
+                                <div class="col-sm-4">
+                                    <div class="clearfix form-group">
+                                        <div class="label-div">
+                                            <label class="control-label" for="iuname">Initial User's Last Name:</label> <a href="#iuname_info"  class="info-anchor icon-tooltip"  data-toggle="collapse" ><i class="fa fa-question-circle" aria-hidden="true"></i></a>
+                                        </div>
+                                        <div>
+                                            <input name='iuname' id='iuname' type='text' class='form-control' value='Administrator'>
+                                        
+                                        </div>
+                                    </div>
+                                    <div id="iuname_info" class="collapse">
+                                        <a href="#iuname_info" data-toggle="collapse" class="oe-pull-away"><i class="fa fa-times oe-help-x" aria-hidden="true"></i></a>
+                                        <p>This is the Last name of the 'initial user'.
+                                    </div>
+                                </div>                  
+                                <div class="col-sm-4">
+                                    <div class="clearfix form-group">
+                                        <div class="label-div">
+                                            <label class="control-label" for="igroup">Initial Group:</label> <a href="#igroup_info"  class="info-anchor icon-tooltip"  data-toggle="collapse" ><i class="fa fa-question-circle" aria-hidden="true"></i></a>
+                                        </div>
+                                        <div>
+                                            <input name='igroup' id='igroup' class='form-control' type='text' value=''>
```

Comment:
Lets default this to Default (as was previously done) since this "feature" is basically not even used in OpenEMR, however values with space or special chars may break things.

Reply:
I can if you want. I have been using values with spaces in my testing, it did not create any issues

Reply:
What if they don't set anything (ie. leave it blank), which is my biggest concern, since the cloning feature does not allow us to require it to be filles.

Reply:
Done